bedrock/mozorg/templates/mozorg/contribute/tasks/v1/joy-of-coding.html

```diff
@@ -0,0 +1,31 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/tasks/taskview-base.html" %}
+
+{% block page_title %}{{ _('Watch: The Joy of Coding') }}{% endblock %}
+
+{% block taskview_title %}{{ _('Watch The Joy of Coding, then discuss it with other Mozillians!') }}{% endblock %}
+{% block taskview_lead %}{{ _('Anyone can contribute to Firefox by finding and fixing errors that people report via Bugzilla. Watch a demonstration of a Firefox engineer live hacking to see what it’s all about!') }}{% endblock %}
+
+{% block task_content %}
+  <ol class="task-steps">
+    <li id="step_one" class="one">
+      <header>
+        <h3>{{ _('Watch The Intro!') }}</h3>
+        <p>{{ _('<a href="https://twitter.com/mike_conley" rel="external">@mike_conley</a> livehacks on real Firefox bugs while thinking out loud.') }}</p>
+      </header>
+      <div class="step-content">
+        <video id="joc" preload="none" controls poster="{{ static('img/contribute/signup/joc_poster.png') }}" data-action="play" data-task="joyofcoding" data-step="one" data-complete="true" data-variation="1">
+          <source src="{{ static('img/contribute/signup/joc-video.mp4') }}" type="video/mp4" />
```

Comment:
Sounds like a good idea to me. I was not aware of that, no do I know how to get something uploaded there so, your assistance here will be much appreciated. Thanks